Makefile

```diff
@@ -1,11 +1,11 @@
 build :
-	cabal new-build -w ghc-8.4.4 --enable-tests
+	cabal new-build -w ghc-8.6.5 --enable-tests
```

Comment:
It might make sense to factor out all these references to `ghc-8.6.5` into an environment variable of some sort so that it is easier to update in the future.